basic_io/src/ru/mail/track/UserStore.java

```diff
@@ -18,37 +19,37 @@ boolean isUserExist(String name) {
     }
 
     // Добавить пользователя в хранилище
-    void addUser(User user) {
-        assert user != null;
-
-        User searchUserResult = getUser(user.getName(), user.getPass());
+    public boolean addUser(User user) {
+        if (user == null) {
+            System.err.println("null pointer at addUser");
```

Comment:
null pointer at addUser ->System.out.println(String.format( "Failed to add user: %s", user));